docker-compose.yml

```diff
@@ -35,6 +35,7 @@ services:
       # taken from .env
```

Comment:
Is there a reason why you have not added `FRONTEND_URL` to the `.env`?

Reply:
I think it's because he was providing a fallback working for dev :-)

But I think it's better to do as @eric-nguyen-cs propose: put `FRONTEND_URL=http://ui.taxonomy.localhost:8091` in .env (this is the default when working with docker I think).